lib/hypervisor/hv_kvm/monitor.py

```diff
@@ -632,10 +590,20 @@ def _GetBlockDevices(self):
     @return: Info about the virtual disks of the instance.
 
     """
-    self._check_connection()
-    devices = self.Execute("query-block")
+    devices = self.execute_qmp("query-block")
     return devices
 
+  @_ensure_connection
+  def ResizeBlockDevice(self, disk_id: str, new_size: int):
```

Comment:
hasn't this been brought by PR #1731 / commit ceff246e30? ATM it puzzles me why this is in the diff of this PR...

Reply:
When solving a merge conflict, this part was removed... This was added again in 4fc8896dfe73a3bc9dd65973dd56dd42ce0d246c

Reply:
I see, it was accidentally removed and the reinserted again, but a view lines below. Originally it was before `_HasPCIDevice()`, now it is after ` _GetBlockDevices()`. Solved as reordering.

Reply:
fixed